openapi/ga4gh-tool-discovery.yaml

```diff
@@ -617,53 +623,56 @@ definitions:
         description: >-
           A docker registry or a URL to a Singularity registry. Used along with
           image_name to locate a specific image.
-        example: 
-         - registry.hub.docker.com
+        example:
+          - registry.hub.docker.com
       image_name:
         type: string
         description: Used in conjunction with a registry_url if provided to locate images
         example:
-         - quay.io/seqware/seqware_full/1.1
-         - ubuntu:latest
+          - quay.io/seqware/seqware_full/1.1
+          - 'ubuntu:latest'
       size:
         type: integer
-        description: >- 
-          Size of the container in bytes
+        description: Size of the container in bytes
       updated:
         type: string
-        description: >-
-          Last time the container was updated.
-      image_type: 
+        description: Last time the container was updated.
```

Comment:
Going beyond the scope of this PR, and it was like this already, and I'm really nitpicking, but sometimes descriptions end with periods and sometimes they don't.